src/client/components/CardProfileResult/CardProfileResultComponent.styles.css

```diff
@@ -1,30 +1,45 @@
-/* * {
-  border: 1px solid red !important;
-} */
-
 .card-profile-result-component {
   background-color: transparent;
-  width: 507px;
+  width: 500px;
 }
 
-h2 {
+.card-profile-result-title {
   font-family: 'Roboto Mono', monospace;
-  font-size: 48px;
-  margin: 30px 0;
+  font-size: 42px;
+  margin-bottom: 64px;
+  color: black;
+  line-height: 55px;
 }
 
 .card-profile-result-list-items {
   background-color: #f0f0f0;
-  width: 400px;
+  height: 420px;
+  width: 440px;
   padding-top: 30px;
   padding-bottom: 30px;
-  padding-right: 50px;
+  padding-right: 100px;
+  padding-left: 70px;
+  list-style: none; /* Need to remove default bullets in order to change their colour */
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
 
-.card-profile-result-list-item {
-  font-family: 'Roboto Mono', monospace;
+.card-profile-result-list-items li::before {
+  content: ' ';
+  width: 14px;
+  height: 14px;
+  background-color: #e95328;
+  display: block;
+  position: absolute;
+  border-radius: 9999px;
+  left: -40px;
+  top: calc(50% - 7px); /* 7px is half the height */
```

Comment:
You have nicely designed the bullet point. But do we not need to use images instead from figma? What do you think?